Document changes to integration test setup in Contrib Guide (#213)
stackabletech/documentation: modules/contributor/pages/steps.adoc

== Changes in Rust projects

1. Make your desired changes in the according repository and test them manually. Ensure that the code compiles without warnings (`cargo clippy --all-targets`) and that the code is formatted with `cargo fmt`.

> 2. If code was added or adapted then please create or adapt the unit tests in the same file as well as the integration tests in the `tests` directory. Ensure that all unit tests run successfully (`cargo test`) and all integration tests run successfully (`./scripts/run_tests.sh`). See also <<_changes_in_the_integration_tests>>.

Reviewer: No "please", please :)
```suggestion
2. If you made source code changes, also update the unit tests in the same file as well as the integration tests in the `tests` directory. Ensure that all unit tests run successfully (`cargo test`) and all integration tests run successfully (`./scripts/run_tests.sh`). See also <<_changes_in_the_integration_tests>>.
```

helm install <operator> deploy/helm/<operator>/
----

> 5. The tests are run with the `scripts/run_tests.sh` script.

Reviewer 2: I would mention that this has to be executed from the repository root folder like ./scripts/run_tests.sh
